CONTRIBUTING.md

```diff
@@ -69,11 +69,9 @@ them!
 ## Release
 
 Let CI pass on GitHub Actions before releasing.
+After you merge your PR, check out develop and pull to get your latest changes.
 The following command will change all action runs to use your current `HEAD`:
-```make release```
-
-### Publishing to pypi
 
 ```
-poetry publish
+make release
```

Comment:
Is `poetry publish` just no longer needed?

Reply:
I think so? That's why I tagged @brendongo for review, too. My memory is that this is not something we do anymore.